mp/src/game/client/momentum/ui/HUD/hud_speedometer_label.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <vgui/IScheme.h>
+#include <vgui_controls/Label.h>
+#include "mom_player_shared.h"
+#include "mom_shareddefs.h"
+#include "utlvector.h"
```

Comment:
I did this with `IScheme`, but was unable to do it with `C_MomentumPlayer` for some reason. It was giving errors whenever I removed the `mom_player_shared.h` include.

Maybe to do with it calling a static method? I've never forward declared a class in this manner before so I'm not sure.

Reply:
What kind of errors?

Reply:
Just an "Incomplete type is not allowed" compiler error. I believe it is because the `C_MomentumPlayer::GetLocalMomPlayer()` function we are using returns a static variable in that file, so the file needs to be imported in order to be able to call it.

Reply:
Yeah, but why does the header need the import? Why not the cpp file?

Reply:
Oh right, I moved that import to the source file now.